client/tsconfig.json

```
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist/",
+    "noImplicitAny": true,
+    "module": "es6",
+    "target": "es5",
+    "jsx": "react",
+    "allowJs": true
+  }
+}
```

Can you add explicit `include` to avoid compiling non-source files? Something like:
```suggestion
  "include": [
    "app"
  ]
}
```